integrations/system/disk-monitoring/README.md

```diff
@@ -0,0 +1,113 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The disk monitoring integration integration provides disk usage monitoring for Linux, Windows, and macOS system.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration provides the following resources:
+
+* "disk-usage" [check]
+* "sensu/check-disk-usage:0.6.0" [asset]
+
+## Dashboards
+
+<!-- List of supported dashboards w/ screenshots (supports png, jpeg, and gif images; relative paths only; e.g. `![](img/dashboard-1.png)` )-->
+
+There are no supported dashboards for this integration.
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
+1. Add one of the following [subscriptions] to [agents] that should run this check.
+
+   * `system`
+   * `system/disk`
+   * `linux`
+   * `windows`
+   * `darwin`
+
+1. Optionally set the `disk_usage_warning_threshold` agent [annotation] to override the default value (`80.0`%).
+
+1. Optionally set the `disk_usage_critical_threshold` agent [annotation] to override the default value (`90.0`%).
+
+## Plugins
+
+<!-- Links to any Sensu Integration dependencies (i.e. Sensu Plugins) -->
+
+This integration does not require any [Plugins].
```

Comment:
This is incorrect,
need to reference the github and bonsai listings for the check plugin here.